src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -190,12 +204,14 @@ public int instanceMain(final String[] argv) {
 
             // Output a one liner about who/where and what software/os we're running on
             try {
-            System.err.println("[" + new Date() + "] Executing as " +
-                                       System.getProperty("user.name") + "@" + InetAddress.getLocalHost().getHostName() +
-                                       " on " + System.getProperty("os.name") + " " + System.getProperty("os.version") +
-                                       " " + System.getProperty("os.arch") + "; " + System.getProperty("java.vm.name") +
-                                       " " + System.getProperty("java.runtime.version") +
-                                       "; Picard version: " + commandLineParser.getVersion());
+                final String msg = String.format(
```

Comment:
Not sure why you changed this to use a variable and `String.format()` instead of using `+`? Do you find this form easier to read?